passageflex/src/main/java/id/passage/passageflex/PassageFlex.kt

```diff
@@ -0,0 +1,139 @@
+package id.passage.passageflex
+
+import android.app.Activity
+import id.passage.android.passageflex.api.AuthenticateAPI
+import id.passage.android.passageflex.api.RegisterAPI
+import id.passage.android.passageflex.model.AuthenticateWebAuthnFinishWithTransactionRequest
+import id.passage.android.passageflex.model.AuthenticateWebAuthnStartWithTransactionRequest
+import id.passage.android.passageflex.model.AuthenticatorAttachment
+import id.passage.android.passageflex.model.RegisterWebAuthnFinishWithTransactionRequest
+import id.passage.android.passageflex.model.RegisterWebAuthnStartWithTransactionRequest
+
+/**
+ *  Use `PassageFlex` to easily add passkeys to your existing authentication system on Apple devices.
+ *
+ *  Find out more at: https://passage.1password.com
+ */
+public object PassageFlex {
+    /**
+     * The base struct for utilizing Apple's native passkey APIs and Passage Flex APIs together.
```

Comment:
Again, I imagine this should be Android?

Reply:
Yes, thank you!